app/pages/lab/visibility.cjsx

```diff
@@ -90,7 +90,7 @@ module.exports = createReactClass
             <p>
               Review status for this project has been approved. To end the review and make changes, switch back to <em>development</em> mode.
               {unless @props.project.launch_requested or @props.project.launch_approved
-                <span>If you’re ready to launch this project, see the next section.</span>}
+                <span>If you’re ready to launch this project, please email contact@zooniverse.org to submit your request.</span>}
```

Comment:
Do you want this to be a working mailto link?

Reply:
I was following Brooke's lead (in PR https://github.com/zooniverse/help/pull/3) to not encode it due to spam harvesting, but seeing that there is a mailto link for this same email on the [About/Contact](https://www.zooniverse.org/about/contact) page, might as well add it.

Reply:
I think bots can scrap email addresses regardless of the presence of an anchor tag using a mailto. I could be wrong so let me do a bit of research of what the current state of this is.

Reply:
Having the address be plaintext in the html whether it is wrapped by an anchor is just as harvestable. Several of the obfuscation techniques break accessibility, so we shouldn't do those. If we're worried about it, then the best solution is probably to setup a webform that has a captcha rather than place the address in the html. At least with the anchor link, it's more convenient for use by the user.

Reply:
Personally, I'm not worried. Contact doesn't seem to get much spam, so might as well make it as easy + accessible as possible.  Thanks for the info!